.gitignore

```diff
@@ -65,3 +65,6 @@ tools/test.file.1G
 
 # autotest logs/data
 /.autotest
+
+# VS Code
+.vscode/
```

Comment:
Don't add this to repository-wide .gitignore. It is specific to your personal dev env, so the correct place for it is in ~/.gitignore on your machine.

Reply:
true. fixed it